File: src/main/java/com/procurement/mdm/MdmApplication.java
package com.procurement.mdm;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.liquibase.LiquibaseAutoConfiguration;
import springfox.documentation.swagger2.annotations.EnableSwagger2;

@SpringBootApplication(exclude = {LiquibaseAutoConfiguration.class})
@EnableSwagger2
public class MdmApplication {
    public static void main(String[] args) {
        SpringApplication.run(MdmApplication.class, args);
    }
}

File: src/main/kotlin/com/procurement/mdm/MdmApplication.kt
package com.procurement.mdm

import com.procurement.mdm.config.ApplicationConfig
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackageClasses = [ApplicationConfig::class])
class MdmApplication

fun main() {
    runApplication<MdmApplication>()
}
